tests/unit/customizations/eks/test_update_kubeconfig.py

```diff
@@ -194,7 +194,7 @@ def setUp(self):
         self._session.create_client.return_value = self._mock_client
         self._session.profile = None
 
-        self._client = EKSClient(self._session, "ExampleCluster", None)
+        self._client = EKSClient(self._session, "ExampleCluster", None, None)
 
```

Comment:
Can we also add a test for when a `--proxy-url` is specified? I imagine it would just take the returned `cluster_entry` and verify that the `proxy_url` set on it is not `None`.